system/jhtree/ctfile.hpp

```diff
@@ -193,11 +193,12 @@ class jhtree_decl CNodeBase : public CWritableKeyNode
 {
 protected:
     NodeHdr hdr;
-    byte keyType;
     size32_t keyLen;
     size32_t keyCompareLen;
     CKeyHdr *keyHdr;
+    byte keyType;
     bool isVariable;
+    std::atomic<bool> ready{false};
```

Comment:
A comment on what this means, and that it is updated only within a critical block, might be useful